daemon/prune.go

```diff
@@ -73,15 +76,24 @@ func (daemon *Daemon) ContainersPrune(ctx context.Context, pruneFilters filters.
 			}
 			cSize, _ := daemon.imageService.GetContainerLayerSize(c.ID)
 			// TODO: sets RmLink to true?
-			err := daemon.ContainerRm(c.ID, &types.ContainerRmConfig{})
-			if err != nil {
-				logrus.Warnf("failed to prune container %s: %v", c.ID, err)
-				continue
-			}
-			if cSize > 0 {
-				rep.SpaceReclaimed += uint64(cSize)
+			if (!dryRunMode) {
```

Comment:
This is duplicated in the `else` block.
I think this can be:

```suggestion
			if !dryRunMode {
				if err := daemon.ContainerRm(c.ID, &types.ContainerRmConfig{}); err != nil {
					logrus.Warnf("failed to prune container %s: %v", c.ID, err)
					continue
				}
			}
			if cSize > 0 {
				rep.SpaceReclaimed += uint64(cSize)
			}
			rep.ContainersDeleted = append(rep.ContainersDeleted, c.ID)
```

*Note*: Don't use GH's UI to accept the change, as the commit won't be signed and it will be rejected by CI.